en/controllers/components.rst

```diff
@@ -52,17 +51,14 @@ also be expressed as::
 
     public function beforeFilter(EventInterface $event)
     {
-        $this->Auth->config('authorize', ['controller']);
-        $this->Auth->config('loginAction', ['controller' => 'Users', 'action' => 'login']);
-
-        $this->Cookie->config('name', 'CookieMonster');
+        $this->RequestHandler->config('viewClassMap', ['rss' => 'MyRssView']);
```

Comment:
There is no ``config()`` method in 4.x anymore, use ``getConfig()`` and ``setConfig()`` methods in code samples and everywhere